cms/djangoapps/contentstore/tests/test_users_default_role.py

```diff
@@ -2,7 +2,6 @@
 Unit tests for checking default forum role "Student" of a user when he creates a course or
 after deleting it creates same course again
 """
-import mock
```

Comment:
Odd, not sure where that came from.. could have sworn I hit an "unused import" violation.. Will add back.